arangod/Aql/CMakeLists.txt

```diff
@@ -194,30 +194,35 @@ target_compile_options(arango_aql PRIVATE
 ## BISON/FLEX
 ################################################################################
 
-#note that we check - in the generated FLEX / BISON files, therefore they are
-#generate inside the source tree
-
-if (USE_MAINTAINER_MODE AND NOT MSVC)
-  set_directory_properties(PROPERTIES CLEAN_NO_CUSTOM "On")
+if (USE_MAINTAINER_MODE)
+  set_directory_properties(PROPERTIES CLEAN_NO_CUSTOM "On") # TODO: Remove this?
 
   add_custom_command(
-    OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/tokens.cpp
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-    COMMAND ${FLEX_EXECUTABLE} -L -otokens.cpp tokens.ll
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/tokens.cpp
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+    COMMAND ${FLEX_EXECUTABLE} -L -otokens.cpp ${CMAKE_CURRENT_SOURCE_DIR}/tokens.ll
     MAIN_DEPENDENCY ${CMAKE_CURRENT_SOURCE_DIR}/tokens.ll
     VERBATIM
   )
 
+  # Find Git Bash on Windows. GITBASH_EXECUTABLE is not set for other OSes
```

Comment:
Works fine under Linux in my test:
```
[ 70%] Generating grammar.cpp
/work/ArangoDB/arangod/Aql/grammar.y:2.1-18: warning: deprecated directive: ‘%name-prefix "Aql"’, use ‘%define api.prefix {Aql}’ [-Wdeprecated]
    2 | %name-prefix "Aql"
      | ^~~~~~~~~~~~~~~~~~
      | %define api.prefix {Aql}
/work/ArangoDB/arangod/Aql/grammar.y: warning: fix-its can be applied.  Rerun with option '--update'. [-Wother]
Consolidate compiler generated dependencies of target arango_aql
[ 70%] Building CXX object arangod/Aql/CMakeFiles/arango_aql.dir/grammar.cpp.o
[ 70%] Linking CXX static library ../../bin/libarango_aql.a
[ 77%] Built target arango_aql
```
It seems that an undefined `${GIBASH_EXECUTABLE}` is simply ignored as intended. If CMake turned this into a call `"" path/utils/bison-c.sh` it would fail, but it seems to fine 👍